src/unary.js

```diff
@@ -30,3 +31,25 @@ export const tan = (input) => unary(input, Math.tan);
 export const copy = (input) => unary(input, (x) => x);
 export const reciprocal = (input) => unary(input, (x) => 1 / x);
 export const sqrt = (input) => unary(input, Math.sqrt);
+export const sign = (input) => unary(input, Math.sign);
+
+export const erf = (input) => {
+  /**
+  *reference 1:https://en.wikipedia.org/wiki/Error_function
```

Comment:
FYI, thanks.

Here we refer to [an "Approximation with elementary function" with maximum error: **1.5×10−7** of **Error function** in Wikipedia](https://en.wikipedia.org/wiki/Error_function) as below figure:


![image](https://github.com/webmachinelearning/webnn-baseline/assets/1531742/3ef3a7b8-3822-4f82-bd84-853291fea368)



and **erf** function of **WebGL backend for TensorFlow.js**.

```js
const ERF = `
  // Error function is calculated approximately with elementary function.
  // See "Handbook of Mathematical Functions with Formulas,
  // Graphs, and Mathematical Tables", Abramowitz and Stegun.
  float p = ${backend_util.ERF_P};
  float a1 = ${backend_util.ERF_A1};
  float a2 = ${backend_util.ERF_A2};
  float a3 = ${backend_util.ERF_A3};
  float a4 = ${backend_util.ERF_A4};
  float a5 = ${backend_util.ERF_A5};

  float sign = sign(x);
  x = abs(x);
  float t = 1.0 / (1.0 + p * x);
  return sign * (1.0 - (((((a5*t + a4)*t) + a3)*t + a2)*t + a1)*t*exp(-x*x));
`;
```